src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -162,6 +159,42 @@ class ProjectTeams extends AsyncView {
     });
   };
 
+  renderAddTeamButton() {
```

Comment:
most of this is just taken from the primary render call. the only thing i added is the jsx label and (soon) a better action button.